app/src/test/java/com/nsc/covidscore/ConverterUniTest.java

```diff
@@ -0,0 +1,18 @@
+package com.nsc.covidscore;
+
+import org.junit.Test;
+
+import java.util.Calendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class ConverterUniTest {
```

Comment:
non-blocker, but this should be ConverterUnitTest

Reply:
Fixed!